src/build/build_implicitconstraints.jl

```diff
@@ -128,19 +195,18 @@ function build_implicitconstraints!(m, jm)
     for s in m._productions
         for input in s.inputs
 
-            ex = :(
-                JuMP.@NLexpression(
-                    $(jm),
-                    $(input.quantity) *
-                    $(y_over_y_bar(jm, s)) *
-                 (       
-                            $(create_cost_expr(jm, s)) * $(get_commodity_benchmark(input.commodity)) /
-                        $(get_jump_variable_for_commodity(jm, input.commodity))
-                )^$(s.elasticity) - 
-                        $(jm[get_comp_demand_name(input)])
-                )
-            )
-
+                ex = :(
+                        JuMP.@NLexpression(
+                            $(jm),
+                            $(input.quantity) *
+                                $(y_over_y_bar(jm, s)) *
+                        (       
+                                    $(create_cost_expr(jm, s)) * $(get_commodity_benchmark(input.commodity)) /
+                                $(get_jump_variable_for_commodity(jm, input.commodity))
+                        )^$(s.elasticity) - 
+                                $(jm[get_comp_demand_name(input)])
+                        )
+                    )
```

Comment:
Does this change anything? Don't think so, right? Would be easier to review if you could keep formatting changes out of the PR.